sdk/storage/azure-storage-queue/HISTORY.md

```diff
@@ -5,6 +5,30 @@
 For release notes and more information please visit
 https://aka.ms/azure-sdk-preview1-python
 
+- **Breaking** New API desgin:
+    - Operations are now scoped to a particular client:
+        - `QueueServiceClient`: This client handles account-level operations. This includes managing service properties and listing the queues within an account.
+        - `QueueClient`: The client handles operations within a particular queue. This includes creating or deleting that queue, as well as enqueuing and dequeuing messages.
```

Comment:
Should we use receiving instead of dequeuing messages to be consistent with the method?

Reply:
It's true the method is called receiving, but "dequeuing" I think consists of both "receiving and deleting" which is what you can do on the QueueClient... so a more "conceptual" description....